### PR TITLE
[Compute] `az vmss`: Add new command `update-domain-walk` to support updating vm in a service fabric vmss

### PR DESCRIPTION
**Pull request by Jing-song**
**Related command**

`az vmss`

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Compute

File: src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_force_recovery_service_fabric_platform_update_domain_walk.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+
+@register_command(
+    "vmss force-recovery-service-fabric-platform-update-domain-walk",
+)
+class ForceRecoveryServiceFabricPlatformUpdateDomainWalk(AAZCommand):
+    """Manual platform update domain walk to update virtual machines in a service fabric virtual machine scale set.
+
+    :example: Force recovery service fabric platform update domain walk
+        az force-recovery-service-fabric-platform-update-domain-walk -g rg --vmss-name vmss --platform-update-domain
```
Comment (yanzhudd) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_force_recovery_service_fabric_platform_update_domain_walk.py:21]:
```suggestion
        az vmss force-recovery-service-fabric-platform-update-domain-walk -g rg --vmss-name vmss --platform-update-domain
```

File: src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_force_recovery_service_fabric_platform_update_domain_walk.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+# --------------------------------------------------------------------------------------------
+
+# pylint: skip-file
+# flake8: noqa
+
+from azure.cli.core.aaz import *
+
+
+@register_command(
+    "vmss force-recovery-service-fabric-platform-update-domain-walk",
```
Comment (yanzhudd) [inline src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_force_recovery_service_fabric_platform_update_domain_walk.py:15]: Is it convient to add a test for this new added command?

Comment (yanzhudd) [reply src/azure-cli/azure/cli/command_modules/vm/aaz/latest/vmss/_force_recovery_service_fabric_platform_update_domain_walk.py:15]: The length of this command is a bit long. It is better to shorten it.

**Comment by zhoxing-ms** [comment]
Could you add some test cases for this new command?

**Comment by Jing-song** [comment]
> Could you add some test cases for this new command?

It is available in cmd, but there are no resources to test it.

**Review by yanzhudd — APPROVED** [review]

**Pull request merged**